syntax = "proto3";
package tinkoff.proto;

// Request
message AddRecipientToDealStepRequest {
  string deal_id = 1;
  string step_id = 2;
  string beneficiary_id = 3;
  float amount = 4;
  optional float tax = 5;
  optional string purpose = 6;
  optional string bank_details_id = 7;
  optional bool keep_on_virtual_account = 8;
}

// Response
message AddRecipientToDealStepResponse {
  string deal_id = 1;
  string step_id = 2;
  string beneficiary_id = 3;
  string recipient_id = 4;
  float amount = 5;
  optional float tax = 6;
  optional string purpose = 7;
  optional string bank_details_id = 8;
  optional bool keep_on_virtual_account = 9;
}
